pkg/daemon/ceph/client/mgr.go

```diff
@@ -179,3 +189,17 @@ func ConfigureBalancerModule(context *clusterd.Context, clusterInfo *ClusterInfo
 
 	return nil
 }
+
+func getMinCompatClientVersion(clusterInfo *ClusterInfo, balancerModuleMode string) (string, error) {
```

Comment:
nit: for readability, this rename would help. I was confused for a minute since it doesn't get actual state
```suggestion
func desiredMinCompatClientVersion(clusterInfo *ClusterInfo, balancerModuleMode string) (string, error) {
```